static/js/dashboard.js

```diff
@@ -420,7 +511,6 @@ $(document).ready(function () {
     pageLength: 10,
     pagingType: "full_numbers",
     scrollY: "503px",
-    scrollCollapse: true,
```

Comment:
This fixes an unrelated bug I found:
If you only have one validator in the dashboard the tooltip of "Last Attestation" has no space and automatically toggles on/off every now and then.